### PR TITLE
Add cuckoo section to example epoch configuration file to use lean30 …

### PR DESCRIPTION
**Pull request by baltar**
…to make it work with default docker container memory limit of 2GB. Also added a note to increase the memory limit to 4GB if one wants to use the default miner.

### CODE REVIEW

**Comment by dincho** [comment]
Yes, let's delay a bit until we sync with upstream, but we should definitely changes the docker defaults or add docker configuration requirements.

**Comment by dincho** [comment]
@lucafavatella what do you think ?

**Comment by lucafavatella** [comment]
> what do you think ?

@dincho 
* This PR makes sense to me.
* I agree on waiting #1748 to be merged first.
* Re "we should definitely change the docker defaults or add docker configuration requirements.": I did not find further places where the Docker config is specified (apart from the `docs/docker.md` amended in this PR) so I do not see further changes needed (apart from rebase after #1748 merged). Please let me know if I am missing anything or misunderstanding your sentence.

**Comment by dincho** [comment]
> * Re "we should definitely change the docker defaults or add docker configuration requirements.": I did not find further places where the Docker config is specified (apart from the `docs/docker.md` amended in this PR) so I do not see further changes needed (apart from rebase after #1748 merged). Please let me know if I am missing anything or misunderstanding your sentence.

It means: yes, I agree with this PR :)

File: docs/docker.md [hunk 1]
```diff
@@ -35,8 +35,16 @@ chain:
 mining:
     autostart: true
     beneficiary: "encoded_beneficiary_pubkey_to_be_replaced"
+    cuckoo:
+        miner:
+            executable: lean30
```
Comment (hanssv) [inline docs/docker.md]:
```suggestion
            executable: lean29-generic
```

File: docs/docker.md [hunk 2]
```diff
@@ -35,8 +35,16 @@ chain:
...
+            extra_args: ""
+            node_bits: 30
```
Comment (hanssv) [inline docs/docker.md]:
```suggestion
            edge_bits: 29
```

File: docs/docker.md (hunk 1 above)
Comment (hanssv) [reply docs/docker.md]: If avx2 is available use `lean29-avx2`

Comment (baltar) [reply docs/docker.md]: Since I don't really know about those miners, I'll just apply the suggestion.

**Pull request merged**